app/javascript/topNavigation/__tests__/utilities.test.js

```diff
@@ -134,4 +141,107 @@ describe('top navigation utilitities', () => {
       );
     });
   });
+
+  describe('initializeTouchDevice', () => {
+    const initialMenuHTML = `
```

Comment:
Out of curiosity, since I know very little about JS tests: it looks like we basically duplicate a piece of code here that is also written in frontend itself - is this intentional solution? Is this sustainable?

Reply:
Yes and no 🙂

It's deliberate, but no it's not sustainable/ideal - it's just a stop gap until we can move the tests to E2E tests in Cypress. There's a comment at the top of the test file to explain this (but it was added previously so not obvious in this PR!)